docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+    A template is made up of custom modules and has several files. The main file that drives this template's deployment lives in the Main.tf file:
+
+    ```HCL
+    resource "azurerm_resource_group" "main" {
+        name     = local.app_rg_name
+        location = local.region
+    }
+
+    module "service_plan" {
+        source              = "../../modules/providers/azure/service-plan"
+        resource_group_name = azurerm_resource_group.main.name
+        service_plan_name   = local.sp_name
+    }
+
+    module "app_service" {
+        source                           = "../../modules/providers/azure/app-service"
+        app_service_name_prefix          = local.app_svc_name_prefix
+        service_plan_name                = module.service_plan.service_plan_name
+        service_plan_resource_group_name = azurerm_resource_group.main.name
+        docker_registry_server_url       = local.reg_url
+        app_service_config               = local.app_services
+    }
+    ```
+
+### **Step 6:** Validate Infrastructure Deployed Successfully
+
+If you correctly completed the previous step, you can trust that Azure resources are now living in Azure. However, let's take it a step further and get your eyes on the infrastructure that was just deployed. Seeing the end result gives you the full quick start experience and is something that should be done for all future Cobalt CIT deployments.
+
+1. Login to the Azure Portal.
+1. Search for "Resource Group" to find the Resource Group menu.
+1. Find and Select the name of the Resource Group output from the deployment.
+1. Select the App Service output from the deployment.
+1. Select the "overview" tab.
+1. Wait for the App Service "URL" link to display itself from within the menu and then visit it.
+
+### **Final Step:** Teardown Infrastructure
+
+Tearing down the infrastructure just created is as easy as running a terraform command within the scope of the same Terraform environment and workspace that created it. Gaining access to the scope of the workspace is as easy as navigating to the Cobalt CIT template directory holding a reference to the environment and workspace.
+
+1. From within the Azure portal, visit the Azure blob storage json file (i.e. "az-hw-$USER") which remotely holds the state of your deployed Azure infrastructure and view it's contents. The next step will wipe out the contents of this file and delete the associated Azure infrastructure.
+
+1. Locally, from the az-hello-world directory, execute the following command to teardown your deployment and delete your resources.
+
+    ```bash
+    # Destroy resources and tear down deployment. Only do this if you want to destroy your deployment.
+    terraform destroy
+    ```
+
+1. From within the Azure portal, revisit the Azure blob storage json file (i.e. "az-hw-$USER")  and view it's contents. At this point, this file should have an empty state.
```

Comment:
```suggestion
1. From within the Azure portal, revisit the Azure blob storage json file (i.e. "az-hw-$USER")  and view it's contents. At this point, this file should have an empty state. Now it can be deleted:

```bash
    # delete the workspace artifacts on blob storage - maybe this lives further down
    terraform workspace select default
    terraform workspace delete az-hw-$USER
```
```